README.md

```diff
@@ -25,3 +25,7 @@ This specification is currently implemented in the
 ## License
 
 SPDX-License-Identifier: [CC-BY-SA-4.0](LICENSE)
+
+## Code of Conduct
+
+Trust, partnership, simplicity and passion are our core values we live and breathe in our daily work life and within our projects. Our open-source projects are no exception. We have an active community which spans the globe and we welcome and encourage participation and contributions to our projects by everyone. We work to foster a positive, open, inclusive and supportive environment and trust that our community respects the micro:bit code of conduct. Please see our [code of conduct](https://microbit.org/safeguarding/) which outlines our expectations for all those that participate in our community and details on how to report any concerns and what would happen should breaches occur.
```

Comment:
The rest of this README tries to use a max line length of 80 characters, could you break down this paragraph in lines of that length? Thanks!

Reply:
Will do